async/kafka/option.go

```diff
@@ -53,3 +53,21 @@ func Start(offset int64) OptionFunc {
 		return nil
 	}
 }
+
+// RequiredAcksPolicy option for adjusting how many replica acknowledgements
+// broker must see before responding.
+// 0 doesn't send any response, the TCP ACK is all you get.
+// 1 waits for only the local commit to succeed before responding.
+// -1  waits for all in-sync replicas to commit before responding.
+func RequiredAcksPolicy(ack int16) OptionFunc {
+	return func(c *consumer) error {
+		if ack < -1 {
```

Comment:
maybe a switch statement could make the above clearer?
```
switch ack {
 case -1:
case 0:
case 1:
   set the value
default:
 return error
}
```

Reply:
Review comments have been addressed in the latest commit. I have also moved the OptionFunc in trace/kafka/option.go since this config makes sense for the AsyncProducer and not for the Kafka Consumer.

Reply:
As I understand you might want to investigate a sync producer since you might be interested to wait for every event send in order to see if it has been accepted or not.
If you need something like that we have to implement much more than an option.

Reply:
Thanks for the suggestion! We are currently investigating if this makes sense for our case (trading speed with reliability). Still setting RequiredAcks to -1 is something that sarama's async producer also respects so we still get value from exposing that option.